packages/ovm/test/contracts/execution-manager.executeCall.spec.ts

```diff
@@ -75,6 +75,44 @@ describe('Execution Manager -- Call opcodes', () => {
   })
 
   describe('executeNonEOACall', async () => {
+    it('fails if the provided timestamp is 0', async () => {
+      // Create the variables we will use for setStorage
+      const intParam = 0
+      const bytesParam = '0xdeadbeef'
+      // Generate our tx calldata
+      const calldata = getUnsignedTransactionCalldata(
+        dummyContract,
+        'dummyFunction',
+        [intParam, bytesParam]
+      )
+      const nonce = await executionManager.getOvmContractNonce(wallet.address)
+      const transaction = {
+        nonce,
+        gasLimit: GAS_LIMIT,
+        gasPrice: 0,
+        to: dummyContractAddress,
+        value: 0,
+        data: calldata,
+        chainId: CHAIN_ID,
+      }
+      const signedMessage = await wallet.sign(transaction)
+      const [v, r, s] = ethers.utils.RLP.decode(signedMessage).slice(-3)
+
+      await TestUtils.assertThrowsAsync(async () => {
```

Comment:
This is good. Is there a good check that we can do to prevent false-positives? A brittle check might be checking for `revert` or something, but false-negatives may be better than false-positives. Also fine to say no and just merge

Reply:
Good call! 👍 I [added and used a `assertRevertsAsync` helper](https://github.com/ethereum-optimism/optimism-monorepo/pull/61/commits/4b153b412af6d09ec9070292806d2d33bb27ef56)